apps/crn-server/test/integration/fixtures/types.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  UserCreateDataObject as CommonUserCreateDataObject,
+  TeamCreateDataObject as CommonTeamCreateDataObject,
+  ResearchOutputCreateDataObject as CommonResearchOutputCreateDataObject,
+  GroupDataObject,
+  GroupRole,
+} from '@asap-hub/model';
+
+export type UserCreateDataObject = Omit<CommonUserCreateDataObject, 'labIds'>;
+export type TeamCreateDataObject = CommonTeamCreateDataObject;
+export type ResearchOutputCreateDataObject = Omit<
+  CommonResearchOutputCreateDataObject,
+  | 'teamIds'
+  | 'methodIds'
+  | 'organismIds'
+  | 'environmentIds'
+  | 'keywordIds'
+  | 'labIds'
+  | 'createdBy'
+> & {
+  teams: string[];
+  published: boolean;
+  methods: string[];
+  organisms: string[];
+  environments: string[];
+  keywords: string[];
+  labs: string[];
+  authors: { userId: string }[];
+};
```

Comment:
The types get rewritten a bit here so that the basic structure of the fixture data is as close as possible to the `POST` payload and response.